content/paths/folders__delete_folders_id.yml

```diff
@@ -32,6 +32,17 @@ responses:
       Returns an empty response when the folder is successfully deleted
       or moved to the trash.
 
+  400:
+    description: |-
+      Returns an error if a folder lock is in place to prevent deletion.
```

Comment:
I think the `400` might happen for a few reasons, might it not? We might want to make the message more generic and say one of the reasons would be a folder lock?

Reply:
This has actually just changed. The new error for both PUT and DELETE folder operations will be 403 access_denied_insufficient_permissions - I'll adjust those endpoint docs to remove the 400s.